src/components/Button/LoginButton.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import IconButton from './IconButton';
+
+import onlineIcon from '../../assets/images/online-icon.png';
+import onlineIconWhite from '../../assets/images/online-icon-white.png';
+
+function LoginButton(props) {
+  const { displayText, handleLogin } = props;
```

Comment:
I think it's good practice to add `...rest` in case someone wants to override something.